extension/src/candlestick.rs

```diff
@@ -1302,4 +1030,31 @@ mod tests {
             assert_eq!(expected, output.unwrap());
         });
     }
+
+    #[pg_test]
+    fn candlestick_byte_io() {
+        let state = tick_data_transition_inner(
+            None,
+            Some(100.into()),
+            Some(10.0),
+            Some(1.0),
+            ptr::null_mut(),
+        );
+        let state = tick_data_transition_inner(
+            state,
+            Some(200.into()),
+            Some(1.0),
+            Some(2.0),
+            ptr::null_mut(),
+        );
+
+        let output_buffer = state.unwrap().to_pg_bytes();
+        let expected = [
```

Comment:
Not necessary if we don't have time, but moving forward it would be good if we can annotate this sort of output with an explanation of what it means just to make it clear that the output is correct (see `explicit_aggregate_test` in `frequency.rs` for an example).